src/components/EstimatedLifecycleCost/index.tsx

```diff
@@ -71,6 +71,16 @@ const Phase = ({ formikKey, year, index, values, errors = [] }: PhaseProps) => {
                       value="Operations and Maintenance"
                       inline
                     />
+
+                    <Field
+                      as={RadioField}
+                      checked={values.phase === 'Other'}
+                      id={`BusinessCase-${formikKey}.Year${year}.Phase${index}.other`}
+                      name={`${formikKey}.year${year}.${index}.phase`}
+                      label="Other"
+                      value="Other"
+                      inline
```

Comment:
Ah, something didn't get translated into the AC because the mockups are more "built out" than what we're trying to achieve in this PR.

The radio options should stack.

All you need to to is remove the `inline` prop from all the `Field`'s and remove `inline` from `RadioGroup` too.